src/services/lnd/schema.ts

```diff
@@ -0,0 +1,75 @@
+import * as mongoose from "mongoose"
+
+const Schema = mongoose.Schema
+
+const hopSchema = new Schema({
+  channel: String,
+  channel_capacity: {
+    type: Number,
+    min: 0,
+  },
+  fee: {
+    type: Number,
+    default: 0,
+  },
+  fee_mtokens: String,
+  forward: {
+    type: Number,
+    min: 0,
+  },
+  forward_mtokens: String,
+  public_key: String,
+  timeout: {
+    type: Number,
+    min: 0,
+  },
+})
+
+const pathSchema = new Schema({
+  fee: {
+    type: Number,
+    default: 0,
+  },
+  fee_mtokens: String,
+  hops: [hopSchema],
+  mtokens: String,
+  payment: String,
+  timeout: {
+    type: Number,
+    min: 0,
+  },
+  tokens: {
+    type: Number,
+    min: 0,
+  },
+  total_mtokens: String,
+})
+
+const paymentSchema = new Schema<LnPaymentType>({
+  status: { type: String, enum: ["settled", "failed", "pending"], required: true },
+  confirmedAt: Date,
+  createdAt: Date,
+  destination: String,
+  milliSatsFee: {
+    type: Number,
+    default: 0,
+  },
+  paymentHash: String,
```

Comment:
please add an index to payment hash

Reply:
Done https://github.com/GaloyMoney/galoy/pull/665/commits/4643aa6da1b3f17daf754e164bb07ba51d8dcf95